velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -114,6 +114,23 @@ TEST_F(DateTimeFunctionsTest, weekOfYear) {
   EXPECT_EQ(15, weekOfYear("2013-04-08"));
 }
 
+TEST_F(DateTimeFunctionsTest, unixDate) {
+  const auto unixDate = [&](std::optional<int32_t> value) {
+    return evaluateOnce<int32_t, int32_t>("unix_date(c0)", {value}, {DATE()});
+  };
+
+  EXPECT_EQ(unixDate(util::fromDateString("1970-01-01")), 0);
```

Comment:
There is a lot of repetition here. Consider changing unixDate lambda to take date string as input and re-write this code as 

```
EXPECT_EQ(unixDate(1970-01-02"), 1);
```

Reply:
Updated.